config/migrations/2023/20231116175906-add-new-dienstverlenende-vereniging-s-Lim/20231116175906-add-mock-user-for-dienstverlenende-vereniging-s-Lim.sparql

```diff
@@ -0,0 +1,85 @@
+PREFIX rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX skos:  <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
+
+<http://data.lblod.info/id/bestuurseenheden/682c21df-fe6a-4266-8f97-109e5b5b4ae4>
+  rdf:type <http://data.vlaanderen.be/ns/besluit#Bestuurseenheid> ;
+  <http://mu.semte.ch/vocabularies/core/uuid>   "682c21df-fe6a-4266-8f97-109e5b5b4ae4" ;
+  skos:prefLabel "s-Lim" ;
+  skos:altLabel "" ;
+  <http://data.vlaanderen.be/ns/besluit#classificatie> <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/d01bb1f6-2439-4e33-9c25-1fc295de2e71> ;
+  <http://mu.semte.ch/vocabularies/ext/kbonummer> "0679631191" ;
+  <http://purl.org/dc/terms/identifier> "0679631191" ;
+  <http://data.vlaanderen.be/ns/besluit#werkingsgebied> <http://data.lblod.info/id/werkingsgebieden/4bd8337aa40880e343b7f0ca8495ef4460e4e79d9e697495b179a93f41aba153> ;
```

Comment:
The werkingsgebied will be removed from the migration for the time being, until the organisatie-portaal team finalizes it on their end.